syncthing.te

```diff
@@ -0,0 +1,45 @@
+policy_module(syncthing, 1.0.0)
+
+# define types
+type syncthing_t;
+type syncthing_exec_t;
+userdom_user_application_domain(syncthing_t, syncthing_exec_t)
+
+type syncthing_config_home_t;
+userdom_user_home_content(syncthing_config_home_t)
+
+# file permissions
+fs_getattr_xattr_fs(syncthing_t)
+
+userdom_manage_user_home_content(syncthing_t)
```

Comment:
Syncthing does not have a well-defined default folder for syncing files ala Dropbox. I'll probably be using it to sync ~/Music, others will probably want to sync ~/Documents, still others might sync an entirely different home directory. The shared directory it creates by default is ~/Sync, but most users are going to configure it to sync the directories they actually care about.